apkid/rules/elf/obfuscators.yara

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2017  RedNaga. http://rednaga.io
+ * All rights reserved. Contact: rednaga@protonmail.com
+ *
+ *
+ * This file is part of APKiD
+ *
+ *
+ * Commercial License Usage
+ * ------------------------
+ * Licensees holding valid commercial APKiD licenses may use this file
+ * in accordance with the commercial license agreement provided with the
+ * Software or, alternatively, in accordance with the terms contained in
+ * a written agreement between you and RedNaga.
+ *
+ *
+ * GNU General Public License Usage
+ * --------------------------------
+ * Alternatively, this file may be used under the terms of the GNU General
+ * Public License version 3.0 as published by the Free Software Foundation
+ * and appearing in the file LICENSE.GPL included in the packaging of this
+ * file. Please visit http://www.gnu.org/copyleft/gpl.html and review the
+ * information to ensure the GNU General Public License version 3.0
+ * requirements will be met.
+ *
+ **/
+
+
+rule ollvm_v3_4 : obfuscator
+{
+  meta:
+    description = "Obfuscator-LLVM version 3.4 svn"
+    info = "https://github.com/obfuscator-llvm/obfuscator/wiki"
+    // "Obfuscator-clang version 3.4 (tags/RELEASE_34/final) (based on LLVM 3.4svn)"
+
+  strings:
+    $clang_version = "Obfuscator-clang version 3.4"
```

Comment:
Is this intentionally "Obfuscator-clang" instead of being "Obfuscator-LLVM" like the others? If so, why no generic rule for "Obfuscator-clang"?

Reply:
It is indeed done intentionally. Realize that v3.4 doesnt contain "OLLVM-obfuscator".  I pasted the string to tailor the rule.

Reply:
Why no generic, non-versioned rule for "Obfuscator-clang"? Is it because only one version exists?

Reply:
Wondering whether remove the `svn` from the description. Recall that the v6.0 contains the wording `git` and version and we're not using it at all. Please let me know your opinion on that.

```
[20:21 xxx@yyy ollvm] > apkid tr.apk
[+] APKiD 1.0.0 :: from RedNaga :: rednaga.io
[*] tr.apk!classes.dex
 |-> anti_vm : possible Build.SERIAL check
 |-> compiler : dx
[*] tr.apk!lib/armeabi/libexploit.so
 |-> obfuscator : Obfuscator-LLVM version 3.4 svn
```